sonic-slave-buster/cross-build-arm-python-reqirements.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+PRE_BUILT_TARGET_DIR=$1
+
+declare -a pkgs=("grpcio==1.38.0" "grpcio-tools==1.20.0" "m2crypto==0.36.0" "bitarray==1.5.3" "lxml==4.6.3")
+
+rm -rf $PRE_BUILT_TARGET_DIR && mkdir $PRE_BUILT_TARGET_DIR
+
+for pkg in "${pkgs[@]}"
+do
+    rm -rf wheel-download
+    mkdir -p wheel-download/sources && pip3 download --platform linux_armv7l --no-deps --dest wheel-download $pkg
```

Comment:
`linux_armv7l` is specific to armhf, can you update this script to support arm64?

Reply:
I'll update on Thursday. We are on holidays vacation now. Thanks.

Reply:
Done.